config.json

```diff
@@ -9,11 +9,11 @@
             "type": "amazon-ebs",
             "access_key": "{{user `aws_access_key`}}",
             "secret_key": "{{user `aws_secret_key`}}",
-            "region": "us-east-1",
-            "source_ami": "ami-9711c6fc",
+            "region": "us-west-2",
```

Comment:
No us-west. Defeats speed increases (and fees) for Synapse data transfer (in us-east)